### ISSUE TITLE
Netlify redirects file is specified as an homepage `alternate`

### ISSUE THREAD

**Issue by chalin** [notaryproject/notaryproject.dev#17]
Currently, the homepage contains:

```html
<link rel="alternate" type="text/netlify" href="https://notaryproject.dev/_redirects">
```

The Netlify redirects file shouldn't be listed as an `alternate` to the site's homepage.

Note: The corresponding etcd.io issue is https://github.com/etcd-io/website/issues/437

**Comment by chalin** [comment]
The corresponding etcd.io fix is https://github.com/etcd-io/website/pull/438.

**Comment by toddysm** [comment]
@nate-double-u trying to clean up the issues. Is this something that still needs to be done?

**Comment by chalin** [comment]
Thanks for noticing @toddysm.
Closed by #69
